includes/class-wc-payments-order-success-page.php

```diff
@@ -401,4 +402,108 @@ public function determine_woopay_order_received_verify_known_shoppers( $value )
 
 		return ! $is_within_grace_period;
 	}
+
+	/**
+	 * Add Multibanco payment instructions to the order on-hold email.
+	 *
+	 * @param WC_Order|mixed  $order The order object.
+	 * @param bool|mixed      $sent_to_admin Whether the email is being sent to the admin.
+	 * @param bool|mixed      $plain_text Whether the email is plain text.
+	 * @param WC_Email|string $email The email object.
+	 */
+	public function add_multibanco_payment_instructions_to_order_on_hold_email( $order, $sent_to_admin = false, $plain_text = false, $email = '' ): void {
+		if ( ! $email instanceof WC_Email_Customer_On_Hold_Order || ! $order || $order->get_payment_method() !== 'woocommerce_payments_' . Payment_Method::MULTIBANCO ) {
+			return;
+		}
+
+		$order_service         = WC_Payments::get_order_service();
+		$multibanco_info       = $order_service->get_multibanco_info_from_order( $order );
+		$unix_expiry           = $multibanco_info['expiry'];
+		$expiry_date           = date_i18n( wc_date_format() . ' ' . wc_time_format(), $unix_expiry );
+		$formatted_order_total = $order->get_formatted_order_total();
+
+		if ( $plain_text ) {
+			echo "----------------------------------------\n";
+			echo __( 'Multibanco Payment instructions', 'woocommerce-payments' ) . "\n\n";
+			printf(
+			/* translators: %s: expiry date */
+				__( 'Expires %s', 'woocommerce-payments' ) . "\n\n",
+				$expiry_date
+			);
+			echo '1. ' . __( 'In your online bank account or from an ATM, choose "Payment and other services".', 'woocommerce-payments' ) . "\n";
+			echo '2. ' . __( 'Click "Payments of services/shopping".', 'woocommerce-payments' ) . "\n";
+			echo '3. ' . __( 'Enter the entity number, reference number, and amount.', 'woocommerce-payments' ) . "\n\n";
+			echo __( 'Entity', 'woocommerce-payments' ) . ': ' . $multibanco_info['entity'] . "\n";
+			echo __( 'Reference', 'woocommerce-payments' ) . ': ' . $multibanco_info['reference'] . "\n";
+			echo __( 'Amount', 'woocommerce-payments' ) . ': ' . wp_strip_all_tags( $formatted_order_total ) . "\n";
+			echo "----------------------------------------\n\n";
+		} else {
+			?>
+			<table class="td" cellspacing="0" cellpadding="6" border="1" width="100%">
+				<tbody>
+				<tr>
+					<td class="td">
+						<table cellpadding="6">
+							<tr>
+								<td rowspan="2" style="padding: 0 5px 0 0;'">
```

Comment:
```suggestion
								<td rowspan="2" style="padding: 0 5px 0 0;">
```
Minor comment, but I think there's a rogue apostrophe here.